src/FileRepository.php

```diff
@@ -147,8 +147,20 @@ public function recursiveSearch(string $folder, string $filename): array
             return [];
         }
 
-        $dir      = new \RecursiveDirectoryIterator($folder);
-        $ite      = new \RecursiveIteratorIterator($dir);
+        $ignoreFiles = function ($fileInfo) {
```

Comment:
I think it would be better if these values could come from a config, not everyone will use jetbrains like vscode users.

I don't understand why this is needed, it appears if the files are in the array or not then return true is still triggered?

Reply:
Yes, we can put it in a config. It is needed because our Modules are a git repository of their own and has a .git folder, and we need to exclude this from the traversal or else there will be a performance degradation since the .git folder contains a lot of files